### PR TITLE
4842 Init project

### PR DESCRIPTION
**Pull request by iamanhquan**
https://edu-redmine.framgia.vn/issues/4842

### CODE REVIEW

**Comment by iamanhquan** [comment]
Ready

**Comment by baclv** [comment]
commented
anh test tool thôi nhé, hem có review đâu

**Comment by iamPhong** [comment]
Ready

**Comment by iamanhquan** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.5.3"
+
+gem "rails", "~> 5.2.2"
+gem "mysql2", ">= 0.4.4", "< 0.6.0"
+gem "puma", "~> 3.11"
+gem "sass-rails", "~> 5.0"
+gem "uglifier", ">= 1.3.0"
+gem "coffee-rails", "~> 4.2"
```
Comment (mdmahmudurrahman) [inline Gemfile]: Please remove this gem.

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by iamanhquan** [comment]
Ready

**Comment by mdmahmudurrahman** [comment]
merged

**Pull request merged**